feature/src/main/resources/feature.xml

```diff
@@ -249,103 +393,114 @@ limitations under the License.
+    </feature>
+    
+    <feature name="jclouds-driver-okhttp" description="OkHttp Client driver for jclouds" version="${project.version}" resolver="(obr)">
+        <feature version='${project.version}'>jclouds</feature>
+        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.okio/${okio.bundle.version}</bundle>
+        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.okhttp/${okhttp.bundle.version}</bundle>
+        <bundle>mvn:org.apache.jclouds.driver/jclouds-okhttp/${jclouds.version}</bundle>
     </feature>
 
-    <feature name='jclouds-digitalocean2' description='Components to access DigitalOcean v2' version='${project.version}' resolver='(obr)'>
-        <feature version='${project.version}'>jclouds-compute</feature>
-        <bundle dependency='true'>mvn:org.apache.jclouds.api/oauth/${jclouds.version}</bundle>
-        <bundle>mvn:org.apache.jclouds.provider/digitalocean2/${jclouds.version}</bundle>
+    <feature name="jclouds-driver-slf4j" description="SLF4J driver for jclouds" version="${project.version}" resolver="(obr)">
+        <feature version='${project.version}'>jclouds</feature>
+        <bundle>mvn:org.apache.jclouds.driver/jclouds-slf4j/${jclouds.version}</bundle>
     </feature>
 
-    <feature name='jclouds-google-compute-engine' description='Components to access Google Compute Engine' version='${project.version}' resolver='(obr)'>
+    <feature name="jclouds-driver-sshj" description="SSHJ driver for jclouds" version="${project.version}" resolver="(obr)">
         <feature version='${project.version}'>jclouds-compute</feature>
-        <bundle dependency='true'>mvn:org.apache.jclouds.api/oauth/${jclouds.version}</bundle>
-        <bundle dependency='true'>mvn:org.apache.jclouds.common/googlecloud/${jclouds.version}</bundle>
-        <bundle>mvn:org.apache.jclouds.provider/google-compute-engine/${jclouds.version}</bundle>
+        <feature version='${project.version}'>jclouds-driver-bouncycastle</feature>
+        <!--TODO update ${jzlib.bundle.version}-->
+        <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.jzlib/1.1.3_2</bundle>
+        <bundle dependency='true'>mvn:com.jcraft/jsch.agentproxy.core/0.0.9</bundle>
+        <bundle dependency="true">mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.jsch-agentproxy-sshj/0.0.9_1</bundle>
+        <!--TODO shall we jclouds version instead?-->
+        <!--<bundle dependency='true'>mvn:org.apache.jclouds.karaf.bundles/jsch-agentproxy-jsch/${project.version}</bundle>-->
+        <bundle dependency="true">mvn:net.i2p.crypto/eddsa/0.1.0</bundle>
+        <!--<bundle dependency="true">mvn:com.hierynomus/sshj/0.19.2</bundle>-->
+        <bundle dependency="true">file:///Users/andrea/git/andreaturli/sshj/build/libs/sshj-0.19.2-dev.24+edfb069.jar</bundle>
```

Comment:
Hmmm you removed the "DO NOT MERGE" warning from the PR title, but this seems to be a blocker... Just FYI! :)